src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -465,6 +465,7 @@ class ChatGroupSpeakOrder(str, Enum):
 
 
 TRACE_LIST_DEFAULT_LIMIT = 1000
+TRACE_LOCAL_TO_CLOUD_EXPERIMENT_NAME = "local_to_cloud"
```

Comment:
@0mza987 Please take over this PR

Reply:
New pr:
https://github.com/microsoft/promptflow/pull/2586